other/field-types/checkbox.md

```diff
@@ -2,55 +2,9 @@
 
 A `checkboxes` field presents an array of checkboxes. Its value is an array containing the values of the selected checkboxes.
```

Comment:
Thinking about "array" since it has technical meaning that doesn't match this. It may be worth mentioning that it allows people to select one or more choices.

Reply:
reworded to be less redundant and more clear.